bin/capsule.js

```diff
@@ -1069,12 +1198,8 @@ const deleteWebStack = async (webProjectName) => {
  * @return {void}
  */
 const createCiStack = async (ciprojectName, site_config) => {
-  await createStack(
-    ciprojectName,
-    await getCiTemplate(),
-    site_config
-  );
-}
+  await createStack(ciprojectName, await getCiTemplate(), site_config);
```

Comment:
I'm confused with the indentation, here is being compacted, but the in the commander options section above is being expanded.